app/views/references/_reference.html.erb

```diff
@@ -9,7 +9,9 @@
     <% else %>
       <h4 class="paper-title">
         <%= link_to paper.title, user_reference_path(reference.list.user, reference.list, reference) %>
-        <small>contributed by <%= link_to reference.user.username, user_profile_path(reference.user) %></small>
+        <% if reference.user_id != reference.list.user_id %>
```

Comment:
I'm playing around with some formatting so most of the rest of the stuff besides the route is just to give a general facelift.  For this, i'm generally concerned about overloading the page with text, especially once we have tags for each of the papers and especially if we do end up deciding to go with methodologies and biases.  Plus an other indicators we might want to show up front like reference count.  

My thinking here is to reduce repetitive text and for this, we can have the reader assume any non-marked reference is contributed by the creator of the list.  

Again, these are just putting some thoughts down to visualize and play around with so let's talk through these.